src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -19,7 +20,14 @@ public ClassifyMessageTests(IConfiguration configuration, ITestOutputHelper outp
             _configuration = configuration;
             _output = output;
             _testClient = testClients;
-            _fixture = fixture;
+
+            if (fixture == null)
+            {
+                throw new ArgumentNullException(nameof(fixture));
+            }
+
+            var useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.CurrentCulture);
```

Comment:
Nit McPick:  InvariantCulture would be a bit more predictable.

Reply:
Fixed, thanks